### PR TITLE
web: add a button for opening/closing sidebar

### PR DESCRIPTION
**Pull request by nicks**
Hello @jazzdan, @yuindustries,

Please review the following commits I made in branch nicks/sidebar:

a8d5c27552e6c87a06e87f6b50b78a0ee8552a9f (2019-03-25 18:37:25 -0400)
web: add a button for opening/closing sidebar

### CODE REVIEW

**Comment by nicks** [comment]
Screenshot from 2019-03-25 18-39-08
Screenshot from 2019-03-25 18-39-17

File: web/src/Sidebar.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 import './Sidebar.scss';
 
-class Sidebar extends Component {
+class Sidebar extends PureComponent {
   render() {
-    return (<div className="Sidebar">I'm a sidebar!</div>)
+    let classes = ['Sidebar']
```
Comment (jazzdan) [inline web/src/Sidebar.js]: semicolon?

Comment (nicks) [reply web/src/Sidebar.js]: why?

Comment (jazzdan) [reply web/src/Sidebar.js]: I come from the "semicolons everywhere" school of thought to avoid ambiguity but I don't feel strongly about it.

**Pull request merged**